MAINTAINERS.md

```diff
@@ -38,10 +38,12 @@ Editors are expected to perform the following duties for this repository. The du
 - Review, triage, comment on, and close submitted GitHub Issues.
 - When appropriate, lead/facilitate architectural discussions in the community.
 - When appropriate, lead/facilitate the creation of a registry road map.
-- Maintain the repository [CONTRIBUTING.md](./CONTRIBUTING.md) file and getting started documents to
+- Maintain the repository [CONTRIBUTING] file and getting started documents to
```

Comment:
Guessing this does a link to a file of the same name?

Reply:
Yes…link is a couple of lines down.  It gets changes in the publishing of the gh-pages.